lib/adapters/mongodb.js

```diff
@@ -836,7 +837,7 @@ adapter._updateOneToMany = function(model, relatedModel, resource, reference, fi
       match[relatedModel.pk || "_id"] = resource[reference.path];
       if (resource._tenantId) match._tenantId = resource._tenantId;
 
-      return wrapAsyncCall(relatedModel, relatedModel.findOneAndUpdate, match, associate);
+      return wrapAsyncCall(relatedModel, relatedModel.findOneAndUpdate, match, associate, { new: true });
```

Comment:
What does `{new: true}` change here? Presume it's something to do with [this issue](https://github.com/Automattic/mongoose/issues/2262) but don't quite follow why would it matter for fortune